src/components/tooltip/tooltip.js

```diff
@@ -81,14 +82,20 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
 
     function configureWatchers () {
       scope.$on('$destroy', function() {
-        scope.visible = false;
+        if (scope.visible !== false) {
+          scope.visible = false;
+          $mdUtil.nextTick(checkVisibility);
```

Comment:
Why are you waiting a frame and then calling hideTooltip when the scope has been destroyed?